streams/src/main/java/org/apache/kafka/streams/processor/WallclockTimestampExtractor.java

```diff
@@ -15,10 +15,9 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.streams.processor.internals;
+package org.apache.kafka.streams.processor;
 
 import org.apache.kafka.clients.consumer.ConsumerRecord;
-import org.apache.kafka.streams.processor.TimestampExtractor;
 
 public class WallclockTimestampExtractor implements TimestampExtractor {
```

Comment:
Could also have a cross-reference to `ConsumerRecordTimestampExtractor` that says "If you need event-time semantics, use `ConsumerRecordTimestampExtractor`" or sth like that.